SunEngine.Cli/Program.cs

```diff
@@ -5,37 +5,42 @@ namespace SunEngine.Cli
 {
     public class Program
     {
-        private static readonly InfoPrinter InfoPrinter = new InfoPrinter();
-        private static readonly ServerStartup ServerStartup = new ServerStartup();
+        private static InfoPrinter InfoPrinter = new InfoPrinter();
```

Comment:
why remove readonly flag?

Reply:
For this case, it's not required I believe.
The main reason to remove readonly - to make this class testable. In case it's readonly, we can't override it's value and can't mock implementation of the class. In case we remove readonly - we can override value with Mock and easily test it.
It could look like this (pseudo language):
````
var mockOfInfoPrinter = Mock(InfoPrinter)
Program.InfoPrinter = mockOfInfoPrinter
Program.Main()
calledOnce(mockOfInfoPrinter.print()) //check that method called once.